test/e2e/framework/resource/resources.go

```diff
@@ -175,6 +175,26 @@ func waitForPodsInactive(ps *testutils.PodStore, interval, timeout time.Duration
 	return err
 }
 
+// filterActivePods returns pods that have not terminated.
+func filterActivePods(pods []*v1.Pod) []*v1.Pod {
+	isPodActive := func(p *v1.Pod) bool {
+		return v1.PodSucceeded != p.Status.Phase &&
+			v1.PodFailed != p.Status.Phase &&
+			p.DeletionTimestamp == nil
+	}
```

Comment:
Let's leave this one as a separate function.
It is handy and it is used in some other e2e tests.
So we can get this framework onto staging and add another utility function to the e2epod pkg.

Reply:
sure